hw/xbox/nv2a/nv2a_vsh.c

```diff
@@ -307,28 +305,30 @@ static QString* decode_swizzle(const uint32_t *shader_token,
         w = vsh_get_field(shader_token, swizzle_field);
     }
 
-    if (x == SWIZZLE_X && y == SWIZZLE_Y
-        && z == SWIZZLE_Z && w == SWIZZLE_W) {
+    if (x == SWIZZLE_X && y == SWIZZLE_Y && z == SWIZZLE_Z && w == SWIZZLE_W) {
         /* Don't print the swizzle if it's .xyzw */
         return qstring_from_str(""); // Will turn ".xyzw" into "."
+
     /* Don't print duplicates */
     } else if (x == y && y == z && z == w) {
-        return qstring_from_str((char[]){'.', swizzle_str[x], '\0'});
+        return qstring_from_str((char[]){ '.', swizzle_str[x], '\0' });
     } else if (y == z && z == w) {
-        return qstring_from_str((char[]){'.',
-            swizzle_str[x], swizzle_str[y], '\0'});
+        return qstring_from_str(
+            (char[]){ '.', swizzle_str[x], swizzle_str[y], '\0' });
```

Comment:
I'd undo the break-change. This and the previous change would make the repeating code pattern more obvious.